spec/lucky/select_helpers_spec.cr

```diff
@@ -41,14 +41,14 @@ describe Lucky::SelectHelpers do
   end
 
   it "renders options" do
-    view.render_options(form.company_id, [{"Volvo", 2}, {"BMW", 3}])
+    view.render_options(form.company_id, [{"Volvo", "2"}, {"BMW", "3"}])
```

Comment:
Does this mean it always needs to be `Tuple(String, String)`? I'm just thinking of the example from a query

```crystal
CarQuery.new.all.map do |car|
  {car.make, car.id.to_s}
end
```

Reply:
Nope! It means that it has to be whatever the type is on the attribute. In this case `company_id` is an attribute with the String type. I can change it to Int32 though so this makes more sense

Reply:
Oh, ok. I like that.